src/modules/user/UserResolver.ts

```diff
@@ -0,0 +1,18 @@
+import { Query, Resolver } from "type-graphql";
+import { User } from "../../model/User";
+import { v4 as uuidv4 } from "uuid";
+import { Inject } from "typedi";
+import { IUserProvider, USER_PROVIDER } from "./UserProvider";
+
+@Resolver(User)
+export class UserResolver {
+  constructor(
+    @Inject(USER_PROVIDER) private readonly userProvider: IUserProvider
+  ) {}
+
+  @Query(() => User, { nullable: false })
+  async generateUser(): Promise<User> {
+    const newUuid = uuidv4();
+    return new User(newUuid);
```

Comment:
Müsste der User hier nicht noch in der Datenbank gespeichert werden? Oder machen wir das erst, wenn wir das erste Mal irgendwelche Userdaten vom Frontend bekommen?

Reply:
hab ich vergessen.